test/util/backend_utils.hpp

```diff
@@ -20,187 +20,210 @@
 #include <ie_core.hpp>
 #include <string>
 #include "ngraph/ngraph.hpp"
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/runtime/tensor.hpp"
 
-using namespace std;
-using namespace ngraph;
-using namespace InferenceEngine;
-
-Blob::Ptr fill_blob(SizeVector shape, std::vector<float> data);
+InferenceEngine::Blob::Ptr fill_blob(InferenceEngine::SizeVector shape, std::vector<float> data);
 
 class Handle;
 
-namespace ov_runtime
+namespace ngraph
 {
-    class Backend;
-    class Tensor
+    namespace ov_runtime
     {
-    public:
-        std::vector<float> data;
-        PartialShape shape;
-        ngraph::element::Type type;
-
-        Shape get_shape() { return shape.to_shape(); }
-        PartialShape get_partial_shape() { return shape; }
-        explicit Tensor(ngraph::element::Type type, PartialShape ps)
-            : type(type)
-            , shape(ps)
+        class Backend;
+        class Tensor : public ngraph::runtime::Tensor
         {
-        }
-        explicit Tensor(ngraph::element::Type type, Shape ps)
-            : type(type)
-            , shape(ps)
+        public:
+            std::vector<int8_t> data;
+
+            explicit Tensor(ngraph::element::Type type, PartialShape ps)
+                : runtime::Tensor(std::make_shared<ngraph::descriptor::Tensor>(type, ps, ""))
+            {
+            }
+            explicit Tensor(ngraph::element::Type type, Shape ps)
+                : runtime::Tensor(std::make_shared<ngraph::descriptor::Tensor>(type, ps, ""))
+            {
+            }
+
+            /// \brief Write bytes directly into the tensor
+            /// \param p Pointer to source of data
+            /// \param n Number of bytes to write, must be integral number of elements.
+            void write(const void* p, size_t n) override
+            {
+                const int8_t* v = (const int8_t*)p;
+                if (v == nullptr)
+                    return;
+                data.resize(n);
+                for (size_t i = 0; i < n; ++i)
+                {
+                    data[i] = v[i];
+                }
+            }
+            /// \brief Read bytes directly from the tensor
+            /// \param p Pointer to destination for data
+            /// \param n Number of bytes to read, must be integral number of elements.
+            void read(void* p, size_t n) const override
+            {
+                int8_t* v = (int8_t*)p;
+                if (v == nullptr)
+                    return;
+                for (size_t i = 0; i < n; ++i)
```

Comment:
First assert that `n == data.size()` then:
`std::copy(data.begin(), data.end(), v);`